js/client-chat.js

```diff
@@ -518,6 +518,15 @@
 				Tools.prefs('showbattles', false);
 				return false;
 
+			case 'showunlink':
```

Comment:
What about /packhidden and /unpackhidden? Removed isn't really accurate for what ``|unlink|hide|`` is doing, but using /show and /hide and ending up /hidehidden as a command doesn't sound like a good idea